lib/variable/operations_common.h

```diff
@@ -54,3 +55,12 @@ template <class T, class Op> auto reduce_all_dims(const T &obj, const Op &op) {
 }
 
 } // namespace scipp::variable
+
+namespace scipp {
+template <class Var, class... Dim>
+auto count_finite(const Var &var, Dim &&... dim) {
+  auto count = sum(isfinite(var), dim...);
+  count.setUnit(units::one);
```

Comment:
Isn't this conceptually cleaner?
May be inconvenient though, so I am not saying that it should be changed.
```suggestion
  count.setUnit(units::count);
```

Reply:
This is computing `N` in the denominator of `mean = sum/N`, so `count` is not useful here.